hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/recovery/ReconOmMetadataManagerImpl.java

```diff
@@ -120,4 +122,11 @@ public long getLastSequenceNumberFromDB() {
     }
   }
 
+  /**
+   * Check if OM tables are initialized.
+   * @return if OM Tables are initialized
+   */
+  public boolean isOmTablesInitialized() {
```

Comment:
Missed `@override` notion, and same comment as above related to Javadoc or it can be completely removed.